components/ninjaone/ninjaone.app.mjs

```
@@ -1,11 +1,338 @@
+    async *paginate({
+      fn, params = {}, maxResults = null, ...opts
+    }) {
+      let hasMore = false;
+      let count = 0;
+      let lastId = 0;
+      const newerThan = params.newerThan;
+
+      do {
+        if (lastId) {
+          params.olderThan = lastId;
+          if (params.newerThan) delete params.newerThan;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)